### PR TITLE
86b1u0xkk

### PR DESCRIPTION
**Pull request by Mohamedshazan**
I have created JWT authentication with dummy data, also i comanted some code in the midlwer file .

### CODE REVIEW

File: server/src/app/auth/login/route.ts [hunk 1]
```diff
@@ -1,16 +1,53 @@
...
+//     const headers = new Headers(request.headers);
+//     headers.set("Set-Cookie", "token=authenticated; Path=/; HttpOnly");
+
+//     return NextResponse.redirect(baseUrl!, { status: 303, headers: headers });
+// }
+
+import jwt from 'jsonwebtoken';
+import bcrypt from 'bcryptjs';
+
+const dummyUser = {
```
Comment (ishara-jay) [inline server/src/app/auth/login/route.ts]: are we gonna complete the task with a dummy user?

File: server/src/app/auth/login/route.ts [hunk 2]
```diff
@@ -1,16 +1,53 @@
...
+      status: 400,
+      headers: { 'Content-Type': 'application/json' },
+    });
+  }
+
+  if (
+    username === dummyUser.username &&
+    bcrypt.compareSync(password, dummyUser.password)
+  ) {
+    const token = jwt.sign({ username }, JWT_SECRET, { expiresIn: '1h' });
```
Comment (ishara-jay) [inline server/src/app/auth/login/route.ts]: let's move the expiration time to configuration (.env)

Comment (ishara-jay) [inline server/src/app/auth/login/route.ts]: let's add role type encrypted to the jwt token.

File: server/src/app/auth/login/route.ts (hunk 1 above)
Comment (Mohamedshazan) [reply server/src/app/auth/login/route.ts]: yes

File: server/src/app/auth/login/route.ts (hunk 2 above)
Comment (Mohamedshazan) [reply server/src/app/auth/login/route.ts]: i have move the expiration time to configuration (.env) done

Comment (Mohamedshazan) [reply server/src/app/auth/login/route.ts]: please cheack  now i have add role type encrypted to the jwt token.

**Review by bsaranga — CHANGES_REQUESTED** [review]
Can you refactor this based on the reference implementation I shared on discord?

**Comment by Mohamedshazan** [comment]
> Can you refactor this based on the reference implementation I shared on discord?



> Can you refactor this based on the reference implementation I shared on discord?

that part has been Completed

File: server/package.json
Comment (bsaranga) [inline server/package.json:1]: Remove the following packages:
`
"@types/bcryptjs": "^2.4.6",
    "@types/cookie-parser": "^1.4.7",
    "@types/express": "^4.17.21",
    "@types/jsonwebtoken": "^9.0.6",
    "antd": "^5.20.2",
    "bcrypt": "^5.1.1",
`

**Review by bsaranga — CHANGES_REQUESTED** [review]
Attend to the requested changes.

**Pull request merged**